src/main/java/org/folio/consortia/repository/TenantRepository.java

```diff
@@ -1,15 +1,16 @@
 package org.folio.consortia.repository;
 
+import java.util.UUID;
+
 import org.folio.consortia.domain.entity.TenantEntity;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
-import java.util.UUID;
-
 @Repository
 public interface TenantRepository extends JpaRepository<TenantEntity, String> {
 
   Page<TenantEntity> findByConsortiumId(UUID consortiumId, Pageable pageable);
+  TenantEntity findFirstByName(String tenantId);
```

Comment:
Based on this name convention Spring Data will build query to find by tenantName, not tenantId

Reply:
Yes, the table column name is "name". And i was thinking tenantId stored here